docs/index.md

```diff
@@ -52,12 +52,18 @@ provider "datadog" {
 - `api_key` (String, Sensitive) (Required unless validate is false) Datadog API key. This can also be set via the DD_API_KEY environment variable.
 - `api_url` (String) The API URL. This can also be set via the DD_HOST environment variable, and defaults to `https://api.datadoghq.com`. Note that this URL must not end with the `/api/` path. For example, `https://api.datadoghq.com/` is a correct value, while `https://api.datadoghq.com/api/` is not. And if you're working with "EU" version of Datadog, use `https://api.datadoghq.eu/`. Other Datadog region examples: `https://api.us5.datadoghq.com/`, `https://api.us3.datadoghq.com/` and `https://api.ddog-gov.com/`. See https://docs.datadoghq.com/getting_started/site/ for all available regions.
 - `app_key` (String, Sensitive) (Required unless validate is false) Datadog APP key. This can also be set via the DD_APP_KEY environment variable.
+- `aws_access_key_id` (String, Sensitive) The AWS access key ID. This can also be set via the AWS_ACCESS_KEY_ID environment variable. Required when using `cloud_provider_type` set to `aws`.
+- `aws_secret_access_key` (String, Sensitive) The AWS secret access key. This can also be set via the AWS_SECRET_ACCESS_KEY environment variable. Required when using `cloud_provider_type` set to `aws`.
+- `aws_session_token` (String, Sensitive) The AWS session token. This can also be set via the AWS_SESSION_TOKEN environment variable. Required when using `cloud_provider_type` set to `aws` and using temporary credentials.
+- `cloud_provider_region` (String) The cloud provider region specifier. Ex `us-east-1` for AWS.
+- `cloud_provider_type` (String) The cloud provider type. Valid values are [`aws`]. We will add support for more cloud providers in the future.
```

Comment:
We can't add comments about what will be provided in the future because that's considered commenting on our stock..

```suggestion
- `cloud_provider_type` (String) The cloud provider type. Valid values are [`aws`].
```